packages/api/amplify_api/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.0-next.6
+
+- Minor bug fixes and improvements
```

Comment:
request: Could this have relevant changes?

```
### Breaking Changes
* feat(api)!: custom primary key support for GraphQL model helpers ([#2606](https://github.com/aws-amplify/amplify-flutter/pull/2606))
Fixes

### Fixes
* fix(api): early call of Amplify creates wrong instance of AmplifyClass
* fix(api): write null values in ModelMutations.create() unless owner field ([#2679](https://github.com/aws-amplify/amplify-flutter/pull/2679))

### Features
* feat(api): GraphQL Subscription Where Filter ([#2650](https://github.com/aws-amplify/amplify-flutter/pull/2650))
```

Reply:
Done 👍